From hjk@linutronix.de  Tue Sep 23 15:59:56 2008
From: "Hans J. Koch" <hjk@linutronix.de>
Date: Thu, 18 Sep 2008 23:03:07 +0200
Subject: UIO: Change driver name of uio_pdrv
To: Greg KH <gregkh@suse.de>
Cc: Uwe Kleine-König <ukleinek@strlen.de>, LKML <linux-kernel@vger.kernel.org>
Message-ID: <20080918210306.GC2991@local>
Content-Disposition: inline


The generic UIO platform device driver should be given a unique driver ID and
not just "uio". This is especially important since we now have a similar driver
named uio_pdrv_genirq. Currently, there's no user of this driver in the
mainline kernel.

Signed-off-by: Hans J. Koch <hjk@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/uio/uio_pdrv.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/uio/uio_pdrv.c
+++ b/drivers/uio/uio_pdrv.c
@@ -12,7 +12,7 @@
 #include <linux/uio_driver.h>
 #include <linux/stringify.h>
 
-#define DRIVER_NAME "uio"
+#define DRIVER_NAME "uio_pdrv"
 
 struct uio_platdata {
 	struct uio_info *uioinfo;
